tt_metal/jit_build/build.cpp

```diff
@@ -230,6 +235,10 @@ JitBuildCompute::JitBuildCompute(const JitBuildEnv& env, int which, bool is_fw)
     this->includes_ = env_.includes_ +
         "-I" + env_.root_ + "tt_metal/hw/ckernels/" + env.arch_name_ + "/inc " +
         "-I" + env_.root_ + "tt_metal/hw/ckernels/" + env.arch_name_ + "/llk_lib " +
+        "-I" + env_.root_ + "tt_metal/hw/ckernels/" + env.arch_name_ + "/metal/common " +
+        "-I" + env_.root_ + "tt_metal/hw/ckernels/" + env.arch_name_ + "/metal/llk_io " +
+        "-I" + env_.root_ + "tt_metal/hw/ckernels/" + env.arch_name_ + "/metal/llk_api " +
+        "-I" + env_.root_ + "tt_metal/hw/ckernels/" + env.arch_name_ + "/metal/llk_api/llk_sfpu " +
```

Comment:
I'm not a fan of adding includes paths to everywhere.  I suspect this is "beyond the scope" of this change, but I'd like to see something more like:
hw/ckernels/inc/<dir>/<file>

and source code that does:
#include "<dir>/file"

and then the build is just:
-I hw/ckernels/inc